spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/CAddTable.scala

```diff
@@ -31,20 +31,23 @@ import scala.reflect._
  * @tparam T Numeric type. Only support float/double now
  */
 @SerialVersionUID(7959261460060075605L)
-class CAddTable[T: ClassTag](val inplace: Boolean = false)(
-  implicit ev: TensorNumeric[T]) extends AbstractModule[Table, Tensor[T], T] {
+class CAddTable[T: ClassTag, D: ClassTag](val inplace: Boolean = false)(
```

Comment:
getClassTagNumerics from AbstractModule should be overrided

Reply:
fixed